clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/validators/loadbalancer/UpsertKubernetesLoadBalancerAtomicOperationValidator.groovy

```diff
@@ -58,7 +58,9 @@ class UpsertKubernetesLoadBalancerAtomicOperationValidator extends DescriptionVa
       helper.validateIpv4(ip, "externalIps[$idx]")
     }
 
-    description.clusterIp ? helper.validateIpv4(description.clusterIp, "clusterIp")  : null
+    if (description.clusterIp && description.clusterIp != "None") {
```

Comment:
From my interpretation of the docs, yes.
- "Valid values are "None", empty string (""), or a valid IP address."

https://kubernetes.io/docs/api-reference/v1.6/#servicespec-v1-core